core/src/commonMain/kotlin/com/ustadmobile/core/view/SchoolDetailView.kt

```diff
@@ -2,22 +2,17 @@ package com.ustadmobile.core.view
 
 import com.ustadmobile.lib.db.entities.School
 
-/**
- * Core View. Screen is for SchoolDetail's View
- */
-interface SchoolDetailView : UstadView {
 
-    //Updates school name on view and sets edit gear icon to visible and feature visibility
-    fun setSchool(school: School)
+interface SchoolDetailView: UstadDetailView<School> {
 
     fun setSettingsVisible(visible: Boolean)
 
-    fun setUpTabs(tabs: List<String>)
+    fun setTitle(title: String)
```

Comment:
Let's change this to a var instead of a function so it is consistent with other views.